build.gradle

```diff
@@ -578,56 +578,60 @@ jlink {
 
     // TODO: Remove the following correction to the merged module
     // The module descriptor automatically generated by the plugin for the merged module contained some invalid entries.
-    // Execute ./gradlew suggestMergedModuleInfo and include the suggested directives here.
+    // Execute ./gradlew suggestMergedModuleInfo, sort, strip ";"", comment non-used modules, and include the suggested directives here.
     mergedModule {
+        requires 'com.fasterxml.jackson.annotation'
+        requires 'com.google.gson'
+        // requires 'com.googlecode.javaewah'
+        // requires 'com.sun.xml.txw2'
+        requires 'jakarta.activation'
+        requires 'jakarta.annotation'
+        requires 'jakarta.inject'
+        requires 'jakarta.validation'
+        requires 'jakarta.ws.rs'
+        // requires 'jakarta.xml.bind'
         requires 'java.compiler'
         requires 'java.datatransfer'
         requires 'java.desktop'
+        // requires 'java.instrument'
         requires 'java.logging'
         requires 'java.management'
         requires 'java.naming'
         requires 'java.net.http'
+        // requires 'java.rmi'
         requires 'java.scripting'
         requires 'java.security.jgss'
         requires 'java.security.sasl'
         requires 'java.sql'
         requires 'java.sql.rowset'
         requires 'java.transaction.xa'
-        requires 'java.rmi'
         requires 'java.xml'
+        // requires 'jdk.attach'
+        // requires 'jdk.jdi'
+        // requires 'jdk.jfr'
         requires 'jdk.jsobject'
+        // requires 'jdk.management'
+        requires 'jdk.security.jgss'
         requires 'jdk.unsupported'
         requires 'jdk.unsupported.desktop'
-        requires 'jdk.security.jgss'
         requires 'jdk.xml.dom'
-        requires 'com.google.gson'
+        requires 'org.apache.commons.lang3'
+        requires 'org.freedesktop.dbus'
+        requires 'org.glassfish.grizzly'
+        requires 'org.glassfish.grizzly.http'
+        requires 'org.glassfish.grizzly.http.server'
+        // requires 'org.jetbrains.annotations'
         requires 'org.jsoup'
         requires 'org.slf4j'
-        requires 'jakarta.xml.bind'
-        requires 'org.apache.commons.lang3'
-        uses 'org.mariadb.jdbc.credential.CredentialPlugin'
-        uses 'org.mariadb.jdbc.authentication.AuthenticationPlugin'
-        uses 'org.mariadb.jdbc.tls.TlsSocketPlugin'
-        uses 'org.mariadb.jdbc.LocalInfileInterceptor'
-        uses 'org.eclipse.jgit.transport.SshSessionFactory'
+        requires 'org.yaml.snakeyaml'
+        uses 'com.sun.webkit.plugin.PluginHandler'
+        uses 'org.controlsfx.glyphfont.GlyphFont'
+        uses 'org.eclipse.jgit.lib.GpgSignatureVerifierFactory'
         uses 'org.eclipse.jgit.lib.GpgSigner'
-        provides 'org.mariadb.jdbc.tls.TlsSocketPlugin' with 'org.mariadb.jdbc.internal.protocol.tls.DefaultTlsSocketPlugin'
```

Comment:
Please revert this commit, this will not work! the merged module have been fine tuned manually to be as small as possible. This will produce runtime exceptions otherwise